### PR TITLE
Adds megaphone icon

### PR DESCRIPTION
**Pull request by akleiner2**
PDS-approved and all.

image

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 3eff5f77d9fd5a6cfd40bf121ea21d901326e55a

https://deploy-preview-281--gestalt.netlify.com

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 1b00e1ee83826b3749d83ce337bb502f6f6fb96d

https://deploy-preview-281--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :file_folder: Bundlesize - <i>Bundle size increase detected, please review</i>
  </td>
    </tr>
  </tbody>
</table>


Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
gestalt.js | :small_red_triangle: 0.2% | :small_red_triangle: 0.4% | 255.87 KB | 256.47 KB | 56.93 KB | 57.18 KB
gestalt.es.js | :small_red_triangle: 0.2% | :small_red_triangle: 0.5% | 253.22 KB | 253.82 KB | 56.55 KB | 56.81 KB
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/gestalt/src/icons/megaphone.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><path fill="#B5B5B5" d="M14,17.499858 C14,18.877858 12.878,19.999858 11.5,19.999858 C10.122,19.999858 9,18.877858 9,17.499858 L9,16.999858 L14,16.999858 L14,17.499858 Z M22.947,15.629858 L18.701,2.71185796 C18.483,2.04985796 17.697,1.78885796 17.135,2.19085796 L1.435,13.430858 C1.065,13.694858 0.91,14.174858 1.053,14.609858 L1.604,16.287858 C1.744,16.712858 2.136,16.999858 2.578,16.999858 L7,16.999858 L7,17.499858 C7,19.984858 9.015,21.999858 11.5,21.999858 C13.985,21.999858 16,19.984858 16,17.499858 L16,16.999858 L21.973,16.999858 C22.673,16.999858 23.168,16.303858 22.947,15.629858 Z"/></svg>
```
Comment (christianvuerings) [inline packages/gestalt/src/icons/megaphone.svg]: @akleiner2 could you ask @jaymarsh to run this through the SVGO optimizer?

image

**Pull request merged**